src/main/java/com/notably/logic/parser/suggestion/NewSuggestionCommandParser.java

```diff
@@ -22,6 +23,10 @@
     private static final String RESPONSE_MESSAGE = "Create a new note";
     private static final String RESPONSE_MESSAGE_WITH_TITLE = "Create a new note titled \"%s\".";
     private static final String RESPONSE_MESSAGE_WITH_TITLE_AND_OPEN = "Create a new note titled \"%s\" and open it.";
+    private static final String ERROR_MESSAGE = "\"%s\" is an invalid command format. "
```

Comment:
Sorry nitpicking here. `"\"%s\" is an invalid creation format. "` better?

Reply:
Ok!